framework/encode/vulkan_state_tracker.cpp

```diff
@@ -1424,5 +1469,129 @@ void VulkanStateTracker::DestroyState(SwapchainKHRWrapper* wrapper)
     }
 }
 
+void VulkanStateTracker::DestroyState(DeviceMemoryWrapper* wrapper)
+{
+    assert(wrapper != nullptr);
+    wrapper->create_parameters = nullptr;
+
+    const auto& entry = device_memory_addresses_map.find(wrapper->address);
+    if (entry != device_memory_addresses_map.end())
+    {
+        device_memory_addresses_map.erase(entry);
+    }
+}
+
+void VulkanStateTracker::DestroyState(AccelerationStructureKHRWrapper* wrapper)
+{
+    assert(wrapper != nullptr);
+    wrapper->create_parameters = nullptr;
+
+    const auto& entry = as_device_addresses_map.find(wrapper->address);
+    if (entry != as_device_addresses_map.end())
+    {
+        as_device_addresses_map.erase(entry);
+    }
+}
+
+void VulkanStateTracker::TrackTlasToBlasDependencies(uint32_t               command_buffer_count,
+                                                     const VkCommandBuffer* command_buffers)
+{
+    if (!command_buffer_count || !command_buffers)
+    {
+        return;
+    }
+
+    for (uint32_t c = 0; c < command_buffer_count; ++c)
+    {
+        CommandBufferWrapper* cmd_buf_wrapper = GetWrapper<CommandBufferWrapper>(command_buffers[c]);
+
+        for (auto& tlas_build_info : cmd_buf_wrapper->tlas_build_info_map)
+        {
+            // Find to which device memory this address belongs to
```

Comment:
You can delete the last `to` because you already said `to which`

Reply:
fixed